tests/vmi_hook_sidecar_test.go

```diff
@@ -265,13 +265,15 @@ func getHookSidecarLogs(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	return string(logsRaw)
 }
 
+// TODO use libvmi package instead
```

Comment:
Nit: Please drop comments.

Reply:
This was a WIP comment that has been removed in a later commit.